lib/travis_event/listener.rb

```diff
@@ -0,0 +1,52 @@
+module TravisEvent
+  def self.handlers
+    @handlers ||= begin
+      workers_path = Rails.root.join("app/workers")
+      Dir.glob(workers_path.join("travis_event_handlers/*.rb")).collect do |f|
+        path = Pathname.new(f).relative_path_from(workers_path).to_s
+        path.chomp(".rb").classify.constantize
+      end
+    end
+  end
+
+  class Listener
+    ALL_EVENTS = %w(build:created build:started build:finished job:created job:started job:finished)
+
+    # repos: ["rails/rails", "manageiq/manageiq"]
+    def self.monitor(*slugs)
```

Comment:
Now accepts an array arg or several repos in the format of the "slug" (not the Travis::Repository instances).